esmgrids/mom_grid.py

```diff
@@ -88,9 +88,8 @@ def fromfile(cls, h_grid_def, v_grid_def=None, mask_file=None,
                 dy_ue = dy_ext[1::2, 3::2] + dy_ext[2::2, 3::2]
 
                 angle_dx = f.variables['angle_dx'][:]
-                # The angle of rotation is a corner cell centres and applies to
-                # both t and u cells.
-                angle_t = angle_dx[2::2, 2::2]
+                
```

Comment:
I think it would be add a comment why we are changing the index to 1::2, 1::2

Reply:
I don't know how to write a concise comment to say it ... The rest of the file is uncommented so i just followed that style !